app/services/resource_recommendations.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class ResourceRecommendations
+  def initialize(moment)
+    @moment = moment
+    @moment_keywords = []
+  end
+
+  def call
+    @moment_keywords = MomentKeywords.new(@moment).call
+    all_resources.select do |resource|
+      tags = resource['tags'].flat_map do |tag|
+        tag.tr('_', '-')
+      end
+      (tags & @moment_keywords).any?
+    end
+  end
+
+  def matched_tags
+    @moment_keywords = MomentKeywords.new(@moment).call
+     resource_tags = all_resources.flat_map do |resource|
+       resource['tags'].map do |tag|
+         tag.tr('_', '-')
+       end
+     end
+     matched_tags = (@moment_keywords & resource_tags)
```

Comment:
Should we be returning `@moment_keywords & resource_tags` instead of setting it to the `matched_tags` variable?